releases/v20.2.0-beta.2.md

```diff
@@ -0,0 +1,103 @@
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.0-beta.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.2.0-beta.2
+~~~
+
+
+### Security updates
+
+- A new experimental flag `--accept-sql-without-tls` has been introduced for [`cockroach start`](cockroach-start.html) and [`cockroach-start-single-node`](cockroach-start-single-node.html). When specified, a secure node will also accept secure SQL connections without TLS. When this flag is enabled:
+
+  - Node-to-node connections still use TLS: the server must still be started with `--certs-dir` and valid [TLS cert configuration](authentication.html) for nodes. 
+  - Client [authentication](authentication.html) (spoof protection) and [authorization](authorization.html) (access control and privilege escalation prevention) are performed by CockroachDB as usual, subject to the HBA configuration (for AuthN) and SQL privileges (for AuthZ). 
+  - Transport-level security (integrity and confidentiality) for client connections must then be provided by the operator outside of CockroachDB—for example, by using a private network or VPN dedicated to CockroachDB and its client app(s). 
+  - The flag only applies to the SQL interface. TLS is still required for the HTTP endpoint (unless `--unencrypted-localhost-http` is passed) and for the RPC endpoint.
+
+		To introduce this feature into an existing cluster: 
+
+		<ol><li>Ensure the cluster ugprade is finalized.</li>
+		<li>Set up the HBA configuration to reject `host` connections for any network other than the one that has been secured.</li>
+		<li>Add the command-line flag and restart the nodes. Note that even when the flag is supplied, clients can still negotiate TLS and present a valid TLS certificate to identify themselves (at least under the default HBA configuration). Finally, this flag is experimental and its ergonomics will likely change in a later version. <a href="https://github.com/cockroachdb/cockroach/pull/54198">#54198</a></li></ol>
```

Comment:
I think the last sentence (Finally, ..) is not a part of the numbered or bullet list. It should be an independent para.